compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -128,6 +128,40 @@ impl<'hir> Iterator for ParentOwnerIterator<'hir> {
     }
 }
```

Comment:
Could you add a FIXME comment to `struct Map` above, saying that the structure was necessary in the past but now it only serves as "namespace" for HIR-related methods, and can be removed if all the methods are reasonably renamed and moved to `tcx` (https://github.com/rust-lang/rust/pull/118256#issuecomment-1826442834).

Reply:
Added in 9509ff4d8089c879efb75accb409fa331a6804fe